moto/medialive/urls.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import unicode_literals
+from .responses import MediaLiveResponse
+
+url_bases = [
+    "https?://medialive.(.+).amazonaws.com",
+]
+
+
+response = MediaLiveResponse()
+
+
+url_paths = {
```

Comment:
You have to add `url_paths` for *all* possible request uris that you have mocked.  The ones you've added only cover a `POST` to either of those paths, but you also need `/prod/channels/<channel-id>`, `/prod/channels/<channel-id>/start`, etc. 

Example: https://github.com/spulec/moto/blob/master/moto/managedblockchain/urls.py